.github/workflows/docker.yml

```
@@ -29,16 +31,16 @@ jobs:
         uses: docker/login-action@v3
         with:
           registry: ghcr.io
-          username: ${{ github.repository_owner }}
+          username: karurochori
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build and push by digest
+      - name: Build and push
         id: build
         uses: docker/build-push-action@v6
-        env:
-          REGISTRY_IMAGE: ghcr.io/${{ env.OWNER_LC }}/vs-fltk
         with:
           file: ./docker/Dockerfile
-          platforms: ${{ matrix.platform }}
+          context: ./docker
+          push: true
+          tags: ${{ env.REGISTRY_IMAGE }}:test
```

```suggestion
          tags: ${{ env.REGISTRY_IMAGE }}:build-env
```

Let's remember to change that before this is merged